corehq/apps/data_interfaces/tests/test_scheduling_integration.py

```diff
@@ -1349,6 +1319,7 @@ def test_next_available_monthly_slot(self, utcnow_patch, send_patch):
             self.assertTrue(instance.active)
 
 
+@use_sql_backend
```

Comment:
Do we still need this decorator?

Reply:
I think we do since the form processing code still defaults to couch without this.